### ISSUE TITLE
location reported in the description field in the Calendar generator

### ISSUE THREAD

**Issue by cristianolongo** [opendatahacklab/agenda-unica#61]
In the file generated by RDFEvents2Calendar.php for each event the location field is empty whereas the field description is filled with the location address.

```
BEGIN:VEVENT
DTSTART:20160720T200000Z
DTEND:20160720T200000Z
DTSTAMP:20160811T153341Z
UID:http://opendatahacklab.org/free-agenda/agenda-unica-ct/20160720/C
 hi+non+ha+memoria+non+ha+futuro+-+reading%2C+video+%26+dibattito/
DESCRIPTION:147, Via Caronda, Catania, Italy
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Chi non ha memoria non ha futuro - reading, video & dibattito
TRANSP:OPAQUE
END:VEVENT
```

**Comment by fedefra** [comment]
solved, forgot to update calendar.php

**Comment by cristianolongo** [comment]
fixed at commit 69ae4463e628fef5cf2572cb1d5ef668bf7402da
